ncl/ncl_index/ncl-index-table.csv

```diff
@@ -36,3 +36,4 @@ NCL Function,Description,Python Equivalent,Notes
 `sqrt <https://www.ncl.ucar.edu/Document/Functions/Built-in/sqrt.shtml>`__,"Computes the square root of its input","``math.sqrt()`` or ``numpy.sqrt()``",`example notebook <../ncl_entries/general_applied_math.ipynb#sqrt>`__
 `sign_matlab <https://www.ncl.ucar.edu/Document/Functions/Contributed/sign_matlab.shtml>`__,"Mimic the behavior of Matlab's sign function","``numpy.sign()``",`example notebook <../ncl_entries/general_applied_math.ipynb#sign-matlab>`__
 `round <https://www.ncl.ucar.edu/Document/Functions/Built-in/round.shtml>`__,"Rounds a float or double variable to the nearest whole number","``round()`` or ``numpy.round()``",`example notebook <../ncl_entries/general_applied_math.ipynb#decimalplaces-round>`__
+`dewtemp_trh <https://www.ncl.ucar.edu/Document/Functions/Built-in/dewtemp_trh.shtml>`__,"Calculates the dew point temperature given temperature and relative humidity","``geocat.comp.dewtemp()``",`example notebook <../ncl_entries/dewtemp.ipynb>`__
```

Comment:
Should this point to the NCL receipt notebook?

Reply:
Actually none of them do, so I'm pretty sure this is correct as is. Are NCL receipts only visible from the GitHub repository (for those really LOOKING for them)?

Reply:
You can get to the receipts from the website, though it's not obvious. 

If you scroll down to the end of the NCL index, there's a link to the [receipts page](https://ncar.github.io/geocat-applications/ncl/receipts/receipts.html#receipts)

Reply:
Ahh I think I used to see it was there when the table was shorter, but now that it's so long we might want a better solution (of course not in this PR).